### PR TITLE
Added basic snippets for sass language, still need to complete documenta...

### PR DESCRIPTION
**Pull request by Vratyas**
...tion

### CODE REVIEW

**Comment by asainz** [comment]
I think it should be useful to add @extend with placeholders. Otherwhise, it is ready to merge

File: sass/sass-snippet.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
+# Sass
+
+## Info
+
+__Demo__: http://codepen.io/Vratyas/pen/xbqjgX?editors=110
+__Gist__: https://gist.github.com/Vratyas/f20d0477dcc09e810cad
+
+## Description
+
+A set of basic snippets showing the most important aspects of the sass language.
```
Comment (DarkThrone) [inline sass/sass-snippet.md]: Please update the description text. It's using the template description.

**Comment by Vratyas** [comment]
Placeholder notation added to the code snippet.

**Comment by DarkThrone** [comment]
LGTM :+1: ima merge

**Pull request merged**